cpp/ql/src/semmle/code/cpp/dataflow/internal/DataFlowImpl.qll

```diff
@@ -1683,6 +1683,11 @@ private class PathNodeMid extends PathNode, TPathNodeMid {
 
   override Configuration getConfiguration() { result = config }
 
+  pragma[noinline]
+  LocalCallContext getLocalCallContext() {
```

Comment:
This looks potentially problematic as it will introduce a new recursive call (which will introduce more `prev` relations in the pipeline).